src/helm/proxy/clients/aleph_alpha_client.py

```diff
@@ -19,9 +23,24 @@ class AlephAlphaClient(Client):
     TOKENIZE_ENDPOINT: str = "tokenize"
     DETOKENIZE_ENDPOINT: str = "detokenize"
 
+    VALID_TOKENIZERS: List[str] = [
+        "luminous-base",
+        "luminous-extended",
+        "luminous-supreme",
+    ]
+
     def __init__(self, api_key: str, cache_config: CacheConfig):
         self.api_key: str = api_key
         self.cache = Cache(cache_config)
+        self._aleph_alpha_client = AlephAlphaPythonClient(token=api_key)
+        self._tokenizer_name_to_tokenizer: Dict[str, Tokenizer] = {}
+
+    def _get_tokenizer(self, tokenizer_name: str) -> Tokenizer:
+        assert tokenizer_name in self.VALID_TOKENIZERS, f"Invalid tokenizer: {tokenizer_name}"
```

Comment:
`raise ValueError(f"Invalid tokenizer: {tokenizer_name}")`? `assert` is generally used for checking internal invariants, not input validation.